plates/dimensions.py

```diff
@@ -0,0 +1,23 @@
+available_dimensions = ((4, 6), (8, 12), (16, 24))
+
+class Dimensions:
+
+    def __init__(self, number_of_rows, number_of_columns):
+        """Initiate attributes for a plate"""
+        dimension = (number_of_rows, number_of_columns)
+        if dimension not in available_dimensions:
```

Comment:
I wouldn't restrict available dimensions.. You could create a set of `Dimensions` constants to re-use DIMENSIONS96, DIMENSIONS384, etc if you want to re-use them all the time. But I don't see why restrict them..